### ISSUE TITLE
[SCREENREADER, KEYBOARD]: Focus SHOULD return to Edit button that launched the modals when modals are closed

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#6156]
## Description

When all the modals are closed and the user is returned to the Profile page, we need to return focus to the Edit button that opened the modal in the first place.

## Point of Contact

**VFS Point of Contact:** Jennifer, Trevor

## Acceptance Criteria

As a keyboard user, I want to open the modal to edit my information, and when I am finished with the modal have the focus on the Edit button so I can return to where I was in the flow of the Profile page.

## Steps to Recreate

1. Enter `https://staging.va.gov/profile/` in browser
2. Log in as user+56
3. Select the Edit button next to mailing address info
4. Cancel the modal
5. Note that focus isn't set — but it should be on the Edit button that opened the modal

**Comment by Samara-Strauss** [comment]
Assigning to @erikphansen or @bradl3yC but this can hang out in the backlog and be addressed with Profile 2.0 cc/ @mshea0606

**Comment by jenstrickland** [comment]
Validated:
- When valid address is entered and Update clicked, focus goes to the message letting user know the system is updating the address.
- Once that is complete, which is relatively quick, the focus is placed on the Edit button
